packages/react-conformance/src/utils/getCallbackArguments.ts

```diff
@@ -290,7 +290,7 @@ export function getCallbackArguments(
     );
   }
 
-  const propertySignature = findPropertySignature(program, sourceFile, typeName, propertyName);
+  const propertySignature = findPropertySignature(program.getTypeChecker(), sourceFile, typeName, propertyName);
```

Comment:
curious, these tests have been running in the repo for a while now, why was this necessary in _**this**_ PR ?

Reply:
The existing method only inspects for interfaces. With the change to types, the props themselves can be intersection types.  I could inspect the type declarations directly, but I would have to iterate over the types array and then recursively handle any types that were themselves intersection types.  The TypeChecker provides this aggregation of properties into a single list to check.